torch/_inductor/freezing.py

```diff
@@ -135,13 +135,35 @@ def run(self):
         return super().run(initial_env=env)
 
 
+def is_impure(node: torch.fx.node.Node):
```

Comment:
Is it better to do the check in the `ConstantFolder`? Then you only need to check the `dequant` node, not need to worry about other nodes that consume `dequant` output since it is already marked as unknown value.

Reply:
Thanks for the suggestions and changed.